src/mscorlib/ref/mscorlib.cs

```diff
@@ -4473,6 +4473,8 @@ public partial class Dictionary<TKey, TValue> : System.Collections.Generic.IColl
         public Dictionary() { }
         public Dictionary(System.Collections.Generic.IDictionary<TKey, TValue> dictionary) { }
         public Dictionary(System.Collections.Generic.IDictionary<TKey, TValue> dictionary, System.Collections.Generic.IEqualityComparer<TKey> comparer) { }
+        public Dictionary(System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<TKey, TValue>> enumerable) {}
+        public Dictionary(System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<TKey, TValue>> enumerable, System.Collections.Generic.IEqualityComparer<TKey> comparer) {}
```

Comment:
Yeah, this file is no longer used. It doesn't hurt to update it, but it makes no actual functional difference.